google-cloud-error_reporting/lib/google/cloud/error_reporting/error_event.rb

```diff
@@ -215,7 +215,7 @@ def self.from_exception exception
           backtrace = exception.backtrace
           message = exception.message
 
-          unless backtrace.nil?
+          if !backtrace.nil? && !backtrace.first.nil?
             error_location = backtrace.first
```

Comment:
What do you think of casting the value as a string instead of the extra conditional?

```ruby
unless backtrace.nil?
  error_location = String backtrace.first
```

Reply:
Or, casting `backtrace` to an array and checking if it is empty?

```ruby
backtrace = Array exception.backtrace
message = exception.message
  
unless backtrace.empty?
```

Reply:
I think String casting is better than Array casting because message includes exception class.
So I'll fix the PR using String casting.